.browserslistrc

```diff
@@ -0,0 +1,32 @@
+and_chr 71
```

Comment:
This is the same as what's in Neodarwin currently (which is from more than year ago, so is pretty conservative).